index.css

```diff
@@ -788,3 +793,112 @@ input[type="email"]:focus {
     .contact-content { width: 100%; }
     .contact-contet__address { margin: 3px auto; }
 }
+
+
+@media only screen and (max-width: 695px) {
+
+
+    .header { text-align: center; }
+    .header__logo { position: absolute; left: 50%; margin-left: -8px; }
+
+
+    /* This is hamburger menu for mobile phones from: https://codepen.io/erikterwan/pen/EVzeRP */
+
+    .navigation  {
+      display: block;
+      z-index: 1;
+      position: relative;
+      -webkit-user-select: none;
+      user-select: none;
+    }
+
+    .navigation input {
+      display: block;
+      width: 40px;
+      height: 32px;
+      position: absolute;
+      top: -7px;
+      left: -5px;
+      cursor: pointer;
+      opacity: 0; /* hide this */
+      z-index: 3; /* and place it over the hamburger */
+     -webkit-touch-callout: none;
+    }
+
+    .navigation span {
+      display: block;
+      width: 33px;
+      height: 4px;
+      margin-bottom: 5px;
+      position: relative;
+      background: #cdcdcd;
+      border-radius: 3px;
+      z-index: 2;
+      transform-origin: 4px 0px;
+      transition: transform 0.5s cubic-bezier(0.77,0.2,0.05,1.0),
+              background 0.5s cubic-bezier(0.77,0.2,0.05,1.0),
+              opacity 0.55s ease;
+    }
+
+
+    .navigation span:first-child {
+        transform-origin: 0% 0%;
+    }
+
+    .navigation span:nth-last-child(2) {
+        transform-origin: 0% 100%;
+    }
+
+    .navigation input:checked ~ span {
```

Comment:
You can set a class name for the `<span>` element, otherwise - if you introduce a new `<span>` element somewhere there on a later stage - this selector will break.